src/Data/stubs.php

```diff
@@ -1515,8 +1526,8 @@
 		'embed-height' => '320',
 		'flashvars' => 'config=http://www.mehr-schbass.de/videos/xml-config.php?id=$2-embed',
 		'id' => '$2',
-		//'name' => '$3',
-	],
+	//'name' => '$3',
+	]
```

Comment:
Trailing comma here is not supposed to be removed. Also a few other changes in whitespace look bad.
Rest seems fine.

Reply:
Whats the sense of a comma if this is the last item of the array?

The whitespaces are just auto formatted code as configured in your .editorconfig file.

Reply:
SlevomatCodingStandard.Arrays.TrailingArrayComma
For smaller diffs in PRs (the previous line wouldnt show as modified etc).
I am fairly certain a few things went a bit too far. But I guess I can revert those after the fact.

Reply:
Im honestly: This is the first time I saw an array with a trailing comma in PHP code.

I stop using automatic code intendation in your repository.
Would you please create an issue for the oEmbed fallback?